app/controllers/admin/topical_event_featurings_controller.rb

```diff
@@ -56,6 +56,10 @@ def order
     redirect_to polymorphic_path([:admin, @topical_event, :topical_event_featurings]), notice: "Featured items re-ordered"
   end
 
+  def confirm_destroy
+    @topical_event_featuring = @topical_event.topical_event_featurings.find(params[:id])
```

Comment:
Could we dry this out, by adding this to a before_action for confirm_destroy and destroy?